packages/editor-ui/src/components/WorkflowTagsDropdown.vue

```diff
@@ -52,9 +52,6 @@ function handleEsc() {
 function handleBlur() {
 	emit('blur');
 }
-
-// Fetch all tags when the component is created
-void tagsStore.fetchAll();
```

Comment:
Just out of curiosity, is this `void <function call>;` vue specific syntax? Any specific term I could look up? Kinda tough to google :D

Reply:
@CharlieKolb [it's a typescript thing](https://typescript.tv/new-features/the-void-operator-in-typescript-and-javascript/), we use it if we don't need to wait for promises to resolve. But should be used carefully tho.